packages/admin/src/pages/build-application/[applicationId]/error-multiple-editors.page.tsx

```diff
@@ -0,0 +1,93 @@
+import { GetServerSidePropsContext } from 'next';
+import Link from 'next/link';
+import InferProps from '../../../types/InferProps';
+import {
+  getApplicationFormSummary,
+  getLastEditedEmail,
+} from '../../../services/ApplicationService';
+import Meta from '../../../components/layout/Meta';
+import { getSessionIdFromCookies } from '../../../utils/session';
+import { formatDateTimeForSentence } from '../../../utils/dateFormatterGDS';
+
+const getLastEditedDate = async (
+  applicationId: string,
+  sessionCookie: string
+): Promise<string> => {
+  try {
+    const application = await getApplicationFormSummary(
+      applicationId,
+      sessionCookie,
+      false,
+      false
+    );
+    const {
+      audit: { lastUpdated },
+    } = application;
+    return formatDateTimeForSentence(new Date(lastUpdated));
+  } catch (e) {
+    console.log(e);
+    return 'unknown';
+  }
+};
+
+const getLastEditedBy = async (
+  applicationId: string,
+  sessionCookie: string
+): Promise<string> => {
+  try {
+    return (
+      (await getLastEditedEmail(applicationId, sessionCookie)) || 'unknown'
+    );
+  } catch (e) {
+    console.log(e);
+    return 'unknown';
+  }
+};
```

Comment:
There's a generic function `fetchDataOrGetRedirect` we could use for this getServerSideProps which will include this error validation logic to clean things up a bit